app/util/helpers.js

```diff
@@ -1,6 +1,40 @@
+import React, {useState, useEffect} from 'react';
+import debounce from "lodash.debounce";
+
+export function useOnScrollBottom(error, isLoading, allIDs) {
```

Comment:
i think helper is more tends to manipulation/operations (it holds utility functionality), here i can see the react code as well so can we extract the react code and put it somewhere....

Reply:
yeah, moved to custom-hooks